Frontend/ui-driver/src/Screens/SubscriptionScreen/ComponentConfig.purs

```diff
@@ -468,8 +468,13 @@ dueDetailsListState state =
       isSplitPayment : item.isSplit,
       id : item.randomId,
       paymentMode : item.mode,
+      isDue : true,
       scheduledAt : if item.mode == AUTOPAY_REGISTRATION then Just (convertUTCtoISC item.scheduledAt "Do MMM YYYY, h:mm A") else Nothing,
-      paymentStatus : if item.mode == AUTOPAY_REGISTRATION then Just (autoPayStageData.stage) else Nothing
+      paymentStatus : if item.mode == AUTOPAY_REGISTRATION then Just (autoPayStageData.stage) else Nothing,
+      boothCharges : case item.specialZoneRideCount, item.specialZoneAmount of
+                      Just 0, Just 0.0 -> Nothing
```

Comment:
No @kavya-shree-s  If I remove this then it will go to following case
`Just count, Just charges -> Just $ show count <> " " <> getString RIDES <> " x ₹" <> HU.getFixedTwoDecimals (charges / DI.toNumber count) <> " " <> getString GST_INCLUDE`

Which we don't want

Reply:
then we should refactor it and remove using Just 0 or Just 0.0 things. Just 0 should be Nothing